CarbonFootprint/TryCatch/HighwayAccidentSimulator.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace TryCatch
+{
+    public class HighwayAccidentSimulator
```

Comment:
Can we have a better name for this class?

Reply:
updated